datacatalog/samples/tests/test_create_fileset_entry_quickstart.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Google LLC
```

Comment:
Let's put this in a `tests/quickstart/` folder to reflect the sample location. You might need a `tests/quickstart/__init__.py` file as well.

Reply:
done